config/twig_hooks/twig_hooks.yaml

```diff
@@ -0,0 +1,13 @@
+sylius_twig_hooks:
+    hooks:
+        'sylius_shop.base#stylesheets':
+            payplug_styles:
+                template: '@PayPlugSyliusPayPlugPlugin/stylesheets/saved_cards.html.twig'
+                priority: 1
+        'sylius.shop.account.saved_cards.index.subcontent':
+            header:
+                template: '@SyliusShop/Account/SavedCards/Index/_header.html.twig'
```

Comment:
we should move theses templates into @PayPlugSyliusPayPlugPlugin namespace instead of SyliusShop, as we don't override a defined one in SyliusShop